TestProjects/VisualEffectGraph_HDRP/Assets/AllTests/Editor/Tests/VFXDataTests.cs

```diff
@@ -93,31 +93,16 @@ public void CheckDataType()
             Assert.IsInstanceOf<VFXDataParticle>(output.GetData());
         }
 
-        string tempFilePath = "Assets/Temp_vfxTest_Ddata.vfx";
-
-        VFXGraph MakeTemporaryGraph()
-        {
-            if (System.IO.File.Exists(tempFilePath))
-            {
-                AssetDatabase.DeleteAsset(tempFilePath);
-            }
-            var asset = VisualEffectAssetEditorUtility.CreateNewAsset(tempFilePath);
-            VisualEffectResource resource = asset.GetResource(); // force resource creation
-            VFXGraph graph = ScriptableObject.CreateInstance<VFXGraph>();
-            graph.visualEffectResource = resource;
-            return graph;
-        }
-
         [OneTimeTearDown]
         public void CleanUp()
         {
-            AssetDatabase.DeleteAsset(tempFilePath);
+            VFXTestCommon.DeleteAllTemporaryGraph();
         }
 
         [Test]
         public void CheckName_Sharing_Between_Output_Event()
         {
-            var graph = MakeTemporaryGraph();
+            var graph = VFXTestCommon.MakeTemporaryGraph();
```

Comment:
N.B.: Not really related to this PR, I realized this editor test wasn't using the common helper.